### PR TITLE
Add binary proxy docs

### PR DESCRIPTION
**Pull request by Hoverbear**
Adds documentation how to use the binary proxies.

### CODE REVIEW

File: docs/src/using-criticalup/running-tools.rst [hunk 1]
```diff
@@ -40,10 +39,80 @@ version of the tool.
 Locating Tools
 ^^^^^^^^^^^^^^
 
-We can find the true path of a tool for the current toolchain with the ``which`` command:
+It is possible to find the absolute path of a tool for the current toolchain with the ``which``
+command:
```
Comment (skade) [inline docs/src/using-criticalup/running-tools.rst:43]: Is this line break intentional?

File: docs/src/using-criticalup/running-tools.rst [hunk 2]
```diff
@@ -40,10 +39,80 @@ version of the tool.
...
 .. code-block::
 
    cd project
    criticalup which rustc
+
+
+Using the Binary Proxies
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+CriticalUp creates a number of 'binary proxies' which can be used to run the appropriate Ferocene
```
Comment (skade) [inline docs/src/using-criticalup/running-tools.rst:55]:
```suggestion
CriticalUp creates a number of 'binary proxies' which can be used to run the appropriate Ferrocene
```

**Review by skade — COMMENTED** [review]
Small typos and a question

File: docs/src/using-criticalup/running-tools.rst (hunk 1 above)
Comment (Hoverbear) [reply docs/src/using-criticalup/running-tools.rst:43]: Yes, it's over 100 chars otherwise. :(

**Review by amanjeev — APPROVED** [review]

**Comment by Hoverbear** [comment]
bors merge

**Pull request merged**